source/manual/zendesk.html.md

```diff
@@ -23,7 +23,10 @@ relevant and actionable - and start with the oldest by date. If external assista
 line is required, assign them on the ticket and move on to the next ticket (keeping the ticket in the
 2nd line queue).
 
-Ignore 'Low' priority tickets - these have been triaged by 2nd line for GOV.UK product managers to review.
+'Low' priority tickets are for tickets that are:
+* `on-hold` or are waiting to be handed over to another team (unless they are product related, they
+  can be assigned to `3rd Line--GOV.UK Product Requests`)
+* tickets that have been dealt with and can be closed if the user doesn't reply in 5 working days
```

Comment:
This looks good. I'm not sure how it fits with this diagram though, which is what I use to triage tickets: https://docs.google.com/presentation/d/1EotoM2CVtqlnx54Qz5bP7OyIx5c9ji_GptUuymHkBrc/edit

Reply:
Thanks for updating the diagram Leena linked to above - looks good 👍 

Just to check my understanding: until now, 1) when a ticket is waiting on more info from the user, we'd mark it as "pending user response" and keep it in the High queue. And 2) when we solve issues, we 'optimistically close' the ticket with a comment, rather than wait for a user response asking them to _confirm_ it's resolved.

Are we now saying that 1) should be moved to the Low queue? **Or** are we saying that the way I've been doing 2) is incorrect (quite possible!) and that we should always wait for user confirmation before closing the ticket?

Depending on the answer above, I'd tweak the wording to either:

```suggestion
* tickets that have been resolved, now awaiting user confirmation ([can be closed](#tickets-left-pending-without-a-response) if no response in 5 working days)
```

...or:

```suggestion
* tickets that are waiting for a response from the user ([can be closed](#tickets-left-pending-without-a-response) if no response in 5 working days)
```

Reply:
Thanks @ChrisBAshton , that's a really good point. I've personally left tickets as `pending` and wait for the user to confirm it's been resolved, but contradicts [this part of the doc](https://docs.publishing.service.gov.uk/manual/zendesk.html#closing-a-ticket) - `Once you’ve resolved a ticket, click “Submit as Solved”.` I think we should be trying to close tickets as early as possible by default and would help fulfil our [SLAs](https://docs.publishing.service.gov.uk/manual/zendesk.html#service-level-agreements) , what do you think @benjamineskola ?

Reply:
I think closing early makes sense — if we're able to reproduce the issue and have been able to confirm for ourselves that it's fixed, I don't know if it's strictly necessary to wait for the user to respond before closing it. And if it turns out we're mistaken and it wasn't fixed, then it's reopened, no harm done.

If we do need the user to confirm that it's fixed, then that could wait around as `pending`, but that shouldn't be necessary in all cases.